core/domain/src/main/java/com/example/domain/repository/AuthRepository.kt

```
@@ -0,0 +1,4 @@
+package com.example.domain.repository
+
+interface AuthRepository {
+}
```

detekt (1.23.8)

[warning] The class or object AuthRepository is empty.

(detekt.empty-blocks.EmptyClassBlock)